### PR TITLE
Add to ES protocol option

### PR DESCRIPTION
**Pull request by ruslanpetrov**
This option also allows you to connect to Elasticsearch over https.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Heavily Inspired by the Realtime Database implementation (Flashlight) by the [Fi
 ## Getting Started:
 
 - Install and run [Elasticsearch](https://www.elastic.co/) either locally or with a service
-- `git clone https://github.com/acupajoe/elasticstore`
+- `git clone https://github.com/ruslanpetrov/elasticstore`
```
Comment (acupajoe) [inline README.md]: Can you change this back to the `acupajoe/elasticstore` repo?

Everything else looks great!

Comment (ruslanpetrov) [reply README.md]: Thanks!
Sorry for `git clone https://github.com/ruslanpetrov/elasticstore`, i think, it should it in my repo. I undo it, sure :)

Comment (ruslanpetrov) [reply README.md]: @acupajoe Done! :)

**Comment by acupajoe** [comment]
@all-contributors please add @ruslanpetrov for PRs

**Comment by acupajoe** [comment]
@all-contributors please add @ruslanpetrov for code

**Comment by ruslanpetrov** [comment]
@acupajoe Thank you for adding me to the contributors, fast feedback and thank you very much for this project, you make life better! :)

**Pull request merged**